### DISCUSSION TITLE
Why is there no llvm@10 ?

### DISCUSSION THREAD

**Discussion by melMass** [Homebrew/discussions#1958] [Everyday usage]
Hi,

I need clang 10 but it's missing for some reason:
```ruby
llvm ✔                  llvm@11                 llvm@9                  llvm@8                  llvm@7
```

**Comment by carlocab** [comment]
See Homebrew/homebrew-core#63177, Homebrew/homebrew-core#63316 and Homebrew/homebrew-core#63938.

We only ship maintained versions of software, but often make exceptions when other existing formulae need an earlier version of another one. This is why the older versioned LLVM formulae were created. However, no other formulae needed `llvm@10` when LLVM 11 was released.

If you need clang 10, you can maintain an `llvm@10` formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Reply by melMass** [reply]
I looked at the issues but not the PRs, I went the tap road, thanks 🙏
